styleguide/source/assets/scss/04-templates/_offset-center.scss

```diff
@@ -57,3 +57,7 @@
   }
 }// end article
 
+// Adding a special case for the homepage.
+.joe__homepage.joe__layout--center-offset .joe__page-content {
+  background-color: transparent;
```

Comment:
This value does not match what's on the JOE single page article. The value is `background-color: rgba(255, 255, 255, 0.9);`

Reply:
![How_Can_Clinicians_Catalyze_Investments_to_Improve_Community_Health____Journal_of_Ethics___American_Medical_Association_and_joe-style-guide_](https://user-images.githubusercontent.com/2271747/55506030-17350580-561a-11e9-9e8c-f3fa08b22529.png)

Reply:
The real effect that takes place in homepage (and in articles) is -
```
.joe__page--top {
  background: linear-gradient(rgba(255, 255, 255, 0.75) 0%, #ffffff 100%);
}
```

It overrides the rule
```
background-color: rgba(255, 255, 255, 0.9);
```
(Which I don't know why is even there, since even IE11 doesn't use it)

The rule `background-color: transparent` in the code you highlighted exist in order to override another rule, which otherwise would set white opaque background
```
.joe__layout--center-offset .joe__page-content {
    background-color: #ffffff;
}
```

Please let me know if that clarifies the quetsion.

Reply:
Probably should not use #ffff instead use white or if you have a variable $white